main/php_ini.c

```diff
@@ -49,17 +48,17 @@
 #endif
 
 
-typedef struct _php_extension_lists {
+typedef struct {
```

Comment:
It's better to keep the struct name so that it can show up in pahole etc. Just remove the leading underscore as that's a reserved name